.github/workflows/master.yml

```
@@ -15,86 +15,42 @@ permissions:
    contents: read
 
 jobs:
-   api_check:
-      runs-on: ubuntu-latest
-      if: github.repository == 'kotest/kotest'
-      steps:
-         -  name: Checkout the repo
-            uses: actions/checkout@v4
-            with:
-               fetch-depth: 0
-               ref: ${{ github.event.inputs.branch }}
-
-         -  name: Setup JDK
-            uses: actions/setup-java@v4
-            with:
-               distribution: 'temurin'
-               java-version: '17'
 
-         -  uses: gradle/actions/setup-gradle@v3
+   validate-abi:
+      name: Run ABI validations
+      if: github.repository == 'kotest/kotest'
+      uses: ./.github/workflows/run-gradle.yml
+      with:
+         runs-on: ubuntu-latest
+         ref: ${{ inputs.ref }}
+         task: apiCheck --continue
 
-         -  name: Run api check
-            run: ./gradlew apiCheck
 
-   linux:
-      needs: api_check
-      runs-on: ubuntu-latest
+   validate-all:
+      name: Run all Gradle validations
       if: github.repository == 'kotest/kotest'
+      needs: check-abi
```

```suggestion
      needs: validate-abi
```
Or does `check-abi` exist somewhere else?

---

Thanks! Yes I made a typo.